ui/src/App.js

```diff
@@ -1,27 +1,52 @@
 import React, { Component } from 'react';
 import { Router, Route, Switch } from 'react-router-dom';
-import { FrontSite, BackOffice } from '@routes/components';
-import { BackOfficeRoutes } from '@routes/urls';
+import { connect } from 'react-redux';
+import { fetchUserProfile } from '@authentication/state/actions';
+import { FrontSite, BackOffice, Accounts } from '@routes/components';
+import { AccountsRoutes, BackOfficeRoutes } from '@routes/urls';
 import history from './history';
 import { NotFound } from '@components';
-import { AuthenticationGuard, UnAuthorized } from './authentication/components';
+import { AuthenticationGuard, UnAuthorized } from '@authentication/components';
 import PropTypes from 'prop-types';
 
+class SetUserInfoComponent extends Component {
```

Comment:
I didn't want to create just a file for this component as I have created only because I cannot wrap the base component with redux. I think we could split it after it gains some more functionality, WDYT?